phpBB/language/en/common.php

```diff
@@ -181,6 +181,7 @@
 	'COOKIE_CONSENT_INFO'	=> 'Learn more',
 	'COOKIE_CONSENT_MSG'	=> 'This website uses cookies to ensure you get the best experience on our website.',
 	'COOKIE_CONSENT_OK'		=> 'Got it!',
+	'COOKIE_CONSENT_HREF'	=> 'http://cookiesandyou.com',
```

Comment:
I dont think thats a good idea since they do not have their ssl certificate properly configured.
Also thats their default option, as you can see here
https://github.com/insites/cookieconsent/blob/master/src/cookieconsent.js#L213